<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableWindow</class>
 <widget class="QMainWindow" name="tableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Trebuchet MS</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Таблица</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #fff</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>650</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(228, 107, 134);
border: none;
border-radius: 10px}

QPushButton:hover {
border: 1px solid  rgb(0, 0, 0);
background-color: rgba(228, 107, 134, 0.9)
}

QPushButton:pressed {
border:3px solid  rgb(0, 0, 0);
background-color: rgba(228, 107, 134, 1)
}</string>
    </property>
    <property name="text">
     <string>НАЗАД</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_windowName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ТЕКСТ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
